### ISSUE TITLE
Error message when I press listen

### ISSUE THREAD

**Issue by mhelamb** [alexa/alexa-avs-sample-app#525]
Hey Guys, I get an error message when I try to press the listen button (see the picture). Do anyone know the issue? I tried to talk with mu iPhone headphones, do I need a usb microphone and a separate speaker? I am new to this so if you need any more information just let me know. Thanks in advance
![screen shot 2017-02-09 at 11 20 42](https://cloud.githubusercontent.com/assets/25660518/22779190/0fafc2d8-eeba-11e6-870d-e1493034321b.png)

**Comment by bronco92** [comment]
Can you possibly share how you fixed it? I have the same issue but actually have a USB Microphone plugged in and working.

**Comment by mhelamb** [comment]
@bronco92 Yes I can, I think that I was pretty lucky. I plugged in my USB microphone before I started doing all steps on step 7. And the second time when i tried it worked.  On the Amazon Alexa guide (https://github.com/alexa/alexa-avs-sample-app/wiki/Raspberry-Pi), hope you get it to work. Caroline Dunn also made a guide how to make your own. If you don't get the amazon github guide to work although it nearly exactly the same guide it could be worth watching her video. https://www.youtube.com/watch?v=JNTuoOsXS1k&t=1s
